src/documentation/netex_idfm_to_ntfs.md

```diff
@@ -195,81 +195,86 @@ comment_id | Line/noticeAssignments/NoticeAssignment/NoticeRef | This field is p
 object_type |  | Fixed value `trip`
-comment_id | *noticeAssignments/NoticeAssignment/NoticeRef* | This field is prefixed. 
+comment_id | noticeAssignments/NoticeAssignment/NoticeRef | This field is prefixed. 
 
 ### trip_properties.txt
-In the Line declaration (cf. `lines.txt`), if a node `Line/keyList/keyValue/Key` contains the value `Accessibility`, a `trip_property` will be specified for all the trips of the line.
+In the Line declaration (cf. `lines.txt`), if a node `Line/AccessibilityAssessment` exists, a `trip_property` will be specified for all the trips of the line.
 
 NTFS field | Netex-IDFM element | Mapping rule/Comment
 --- | --- | ---
-trip_property_id | *Line/@id* | The id of the line is used to create this object. This field is prefixed. 
-wheelchair_accessible | `Line/keyList/keyValue/Value` | See (1) below.
+trip_property_id | Line/AccessibilityAssessment/@id | The id of the line is used to create this object. This field is prefixed. 
+wheelchair_accessible | Line/AccessibilityAssessment/MobilityImpairedAccess | See (1) below.
+visual_announcement | Line/AccessibilityAssessment/limitations/AccessibilityLimitation/VisualSignsAvailable | See (1) below.
+audible_announcement | Line/AccessibilityAssessment/limitations/AccessibilityLimitation/AudibleSignalsAvailable | See (1) below.
 
-wheelchair_accessible value: 
-* If source value is "0", then this property is set to "2" (not accessible),
-* If source value is "1", then this property is set to "1" (accessible),
-* else this property is set to "0" (unknown)
+(1) setting accessibility value: 
+* If source value is `false`, then this property is set to `2` (not accessible),
+* If source value is `true`, then this property is set to `1` (accessible),
+* else this property is set to `0` (unknown)
 
 
 ### stop_times.txt
-`stop_times` of a trip are listed in the `passingTimes/TimetabledPassingTime` nodes of a `ServiceJourney` (cf. `trip` definition).
+`stop_times` of a trip are listed in the `ServiceJourney/passingTimes/TimetabledPassingTime[]` nodes. They are positioned in the same order as the `ServiceJourneyPattern/pointsInSequence` corresponding to the ServiceJourney.
 
 NTFS field | Netex-IDFM element | Mapping rule/Comment
 --- | --- | ---
-trip_id | *ServiceJourney/@id* | This field is prefixed. 
+trip_id | ServiceJourney/@id | This field is prefixed. 
 stop_sequence | | Auto-incremented field starting with `0` for the first stop_time
 stop_id | | See (1) below
-arrival_time | *TimetabledPassingTime/ArrivalTime* | If `TimetabledPassingTime/DepartureDayOffset` value is >0, arrival_time is incremented 24 hours for each day offset. 
-departure_time | *TimetabledPassingTime/DepartureTime* | If `TimetabledPassingTime/DepartureDayOffset` value is >0, departure_time is incremented 24 hours for each day offset. 
+arrival_time | TimetabledPassingTime/ArrivalTime | If `TimetabledPassingTime/DepartureDayOffset` value is >0, arrival_time is incremented 24 hours for each day offset. 
```

Comment:
Does this mean we can end up with `25:00` (if we have `1:00` in `ArrivalTime` and a `DepartureDayOffset` to `1`)? Even if no modulo operation is mentioned here, we have a serious tendency to automatically do it when it comes to hours... so it might worth making it explicit here.

Reply:
ok, I've added it